### PR TITLE
Convert ClaimBuster API integration to V2

### LINKED ISSUE #139: Use ClaimBuster v2 API
The ClaimBuster team has a new version of the API and would like us to start using it, preferably by the end of the month. See https://idir.uta.edu/claimbuster-dev/api/ and https://idir.uta.edu/claimbuster-dev/api/docs/. Our API key is in the vault.

We should keep v1 API support available for a while and make API version env-configurable.

### LINKED ISSUE #170: Claimbuster breaks when the statement is wildly long
It seems some CNN transcripts can have really long uninterrupted comments (e.g. in a speech) which our parser treats as a single statement right now.

For [example](http://transcripts.cnn.com/TRANSCRIPTS/1908/19/cg.01.html): 

> Thank you so much to my dear friend and colleague, Congresswoman Ilhan Omar for inviting me to her district to join y'all today.   I'm incredibly thankful for her leadership and strength through all she has been dealing with as a woman of color in Congress.  I don't know how she does it sometimes, but the outpouring of support we have received from our constituents and supporters across the country shows us how important it is to keep fighting for justice.   Today's -- today, Reps Omar and Plaskett and I were supposed to be on a congressional trip and delegation to the Palestine and Israel and such delegations are common occurrence for members of Congress. Earlier this month, in fact, 71 other members of Congress traveled to Israel seemingly without incident.  What is not common occurrence as members of Congress being barred from entering a country on these fact-finding missions unless they agree to strict set of rules curtailing rights or being required to submit their itineraries for stop by stop pre-approval.   History does have a habit of repeating itself.  I learned this week that a former member of Congress, Congressman Charles C. Diggs Jr. was denied entry into apartheid South Africa in 1972.  He was also the representative for the 13th congressional district in Michigan.   I was born and raised in the beautiful Detroit, where many of my African-American teachers taught me about the realities of oppression and justice and the need to speak up and take action.  Growing up in a city that has been at the center of many social justice movements for civil rights, labor rights and equality and absorbing those lessons has shaped who I am today, and drives me to push for peace and justice for the Palestinian and Israeli people.  As a young girl visiting Palestine to see my grandparents and extended family, I watched as my mother had to go through dehumanizing checkpoints -- even though she was a United States citizen and proud American.  I was there when my city was in a terrible car accident and my cousins and I cried so she could have access to the best hospitals which were in Jerusalem.  I remember shaking with fear when check points appeared in the small village of Beit Ur al-Fauqa, tanks and guns everywhere.   I remember visiting East Jerusalem with my then-husband and him escorted off the bus, although he was a United States citizen just so security forces could harass him.  All I can do as my city's granddaughter, as the granddaughter of a woman who lives in occupied territory is to elevate her voice by exposing the truth, the only way I know how.  As my Detroit public schools teachers taught me, by humanizing the pain of oppression, our delegation trip included meetings with Israeli veterans, who were forced to participate in military occupation.  They also desperately want peace and self-determination for their Palestinian neighbors.  They could have shed light into injustices of raids, shootings and demolition and child detention.   The delegation would have seen firsthand why walls are destructive and not productive.  They could have asked the people in Bethlehem how walls cut people off away from economic opportunities from a way to live and do psychological damage that lasts forever.  All I can do as her granddaughter is help humanize her and the Palestinian people's plight.  I know that when we could finally see them as deserving of human dignity, everyone who lives there will be able to live in peace.  It is unfortunate that Prime Minister Netanyahu is apparently taken a page out of Trump's book.  And even direction from Trump to deny this opportunity.  And, yes, while folks are shocked this happened to us, but today we will hear from folks who helped show the reality for many who have been barred from going to Israel, not to be even able to reach the Palestinian people.  They are fellow Americans who cannot visit their families or their loved ones.   They should be -- all of us should be deeply deserved, all Americans should be deeply disturbed and with that I thank you so much to my colleague Congresswoman Ilhan Omar for helping humanize the Palestinian people.

We either need to split these into smaller chunks, or we need to see if claimbuster v2 offers an alternative approach to querying, which will happen with #139.

### LINKED ISSUE #204: Statements with forward slash breaks claimbuster
Claimbuster doesn't like it when you pass in a string with a slash in it (even if that slash is uriencoded.)

We should strip those for now, and also consider letting claimbuster know about the issue.

### LINKED ISSUE #214: Sanitizing statements for ClaimBuster breaks URLs in tweets
Newsletter:

> [Joe Biden (Twitter)](https://twitter.com/JoeBiden/status/1167912444569759744):
> Obamacare covered 20 million additional Americans, protects over 100 million with pre-existing conditions, and allo… https:t.coxZenDTrqH6

That URL is mangled and unusable since we started [stripping slashes out of statements](https://github.com/TechAndCheck/tech-and-check-alerts/commit/0dc0577bcb0834cf0e92cb45f82e6ee5b7d2946b) to resolve issues with ClaimBuster V1 API (#204).

It's possible that problem goes away in the V2 API, but really stripping out slashes is a draconian solution anyway, as it may be critical to the legibility of the statement.

### PR DESCRIPTION
**Pull request by reefdog**
This changes our ClaimBuster integration to interact with V2 of the API.

We now POST the text to the API in a JSON payload rather than just appending the statement to the URL as a GET request. Not only is this more correct, it has the additional benefit of:

1. Resolving the issue with slashes in content (#204), which was due to ClaimBuster’s GET requests failing on encoded slashes (`%2F`), without requiring us to strip them out (#214). Thus, this also removes the now-unnecessary text-cleaning and associated tests.
2. Resolving the issue with really long statements (#170), since POST bodies can be gobs larger than GET requests.

Since the V2 endpoint requires authorization, this also adds a new `.env` variable for the API key that must be set before integration will work.

Resolves #139
Resolves #204
Resolves #214
Resolves #170

_**Note:** This PR replaces #210, which was such a wrong architecture that it wasn't worthwhile resolving it with a commit or even a force-push. I'll be closing that PR and deleting its branch, but feel free to consult it for some conversation about a V1+V2 switchable architecture._

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -42,3 +42,5 @@ TWITTER_OAUTH_TOKEN_SECRET=
 TWITTER_CONSUMER_KEY=
 TWITTER_CONSUMER_SECRET=
 
+# ClaimBuster API settings
+CLAIMBUSTER_API_KEY=ABC123
```
Comment (slifty) [inline .env.template]: We don't have example values for other template envs; lets maintain that pattern (unless there's a reason to include one!)

File: src/server/constants.js [hunk 2]
```diff
@@ -15,3 +13,7 @@ export const PLATFORM_NAMES = {
   CNN: 'CNN',
   TWITTER: 'Twitter',
 }
+
+export const CLAIMBUSTER_THRESHHOLD = 0.5
+
+export const CLAIMBUSTER_API_ROOT_URL = 'https://idir.uta.edu/claimbuster-dev/api/v1'
```
Comment (slifty) [inline src/server/constants.js]: [v1 means v2? what a country](https://www.youtube.com/watch?v=Q8mD2hsxrhQ)

File: src/server/utils/claimBuster.js [hunk 3]
```diff
@@ -1,27 +1,7 @@
-import {
-  runSequence,
-  squish,
-} from '.'
+// Disabling because we intend to have more exports in the future.
```
Comment (slifty) [inline src/server/utils/claimBuster.js]: too soon :(

File: .env.template (hunk 1 above)
Comment (reefdog) [reply .env.template]: Ah yes true!

File: src/server/utils/claimBuster.js (hunk 3 above)
Comment (reefdog) [reply src/server/utils/claimBuster.js]: Maybe this should read "because we had more exports in the past".

File: src/server/constants.js (hunk 2 above)
Comment (reefdog) [reply src/server/constants.js]: Yeah, that one threw me for a loop too.

**Pull request merged**